rpa-manager/modules/ROOT/pages/myrpa-start.adoc

```diff
@@ -10,13 +10,14 @@ You can start the automation of a process as a project manager from your backlog
 
 == Before You Begin
 
-* Ask an administrator to xref:usermanagement-manage.adoc#assign-privileges-to-a-user[assign you the required privileges]:
+* Ask an organization administrator in Access Management to assign you the required permissions:
 +
-[cols="1,2"]
+[cols="1,1,1"]
 |===
-|*Action* |*Required Privileges*
+|*Action* |*RPA Permission* | *Deprecated RPA Permission*
 
 |Start automation of a process.
+|RPA Automations Designer, RPA Automations Contributor, RPA Automations Manager, RPA Administrator
```

Comment:
Would it be better to say?:
"RPA Automations Designer, RPA Automations Contributor, RPA Automations Manager, and RPA Administrator"
If so, make the change globally.